user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_TakeConfigurationBackup.md

```diff
@@ -0,0 +1,174 @@
+
+    ![Select connector](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_1.png)
+
+1. Selected the connector and click the *Generate* button in the interactive automation script
+1. By toggling off the *Show All* button, you can easily locate the generated CI Type
+
+    ![Generate CI Type](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_2.png)
+
+## Step 2: Create an element and manage it
+
+1. Create an element with the [deployed connector](#deploy-connector)
+1. Navigate to *Inventory* > *Unmanaged*
+1. Click on the *Refresh* button to refresh the table
+
+    > [!NOTE]
+    > Make sure the *Detected CI Type* is filled in with the [generated CI Type](#generate-ci-type).
+    >
+    > If not, confirm that the element was created with the production version.
+
+1. Select the recently created element and click on the *Manage* button
+
+    ![Unmanaged element](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_UnmanageElement.png)
+
+1. Confirm the element is now managed by IDP, by navigating to *Inventory* > *Managed*
+
+    ![Managed element](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_ManageElement.png)
+
+## Step 3: Setup configuration archive
+
+For IDP to store the configurations it needs to access a local folder or a shared folder.
+
+1. Open the *DataMiner IDP* app.
+1. Navigate to *Admin* > *Configuration*
+1. Fill the *Dataminer Configuration Archive* settings
+
+    ![Setup configuration archive credentials](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_ConfigurationArchiveSetup.png)
+
+1. Click the *Connect* button
+
+> At this point, we have already
+>
+> 1. generated the CI Type
+> 1. created an element and added it to the managed elements
+> 1. configured IDP to access the archive
+
+In the next step, we will create the script responsible for getting the configuration and sending it to IDP.
+
+## Step 4: Create the script to retrieve the configuration backup
+
+When IDP is installed it comes with an example script on how to retrieve the configuration and send it to IDP. First, we will start by duplicating the example script and, then change it to retrieve the parameter values from the [element created in Step 2](#step-2-create-an-element-and-manage-it).
+
+### Duplicate the example configuration backup script
+
+1. Navigate to *Automation*
+1. Expand the folder until you reach *DataMiner Solutions* > *IDP* > *CI Type Management* > *Configuration Management* > *Backup*
+1. Duplicate the script *IDP_Example_Custom_ConfigurationBackup* and rename it
+
+    ![Duplicated script](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_DuplicateScript.png)
+
+### Update the script
+
+1. Open the script
+1. Navigate to the function *GetDeviceFullBackupAsText*
+1. Change the function to retrieve the values from parameters 502 (*System Contact*) and 504 (*System Location*)
```

Comment:
Add a note here that these are parameters for the suggested connector but if another was chosen that these should be adapted accordingly